### PR TITLE
Home page table cleanup

### PR DESCRIPTION
**Pull request by cthoyt**
This PR makes several improvements to the homepage:

1. Fixes trailing whitespace on truncated descriptions

    | Before | After |
    |:------:|:----:|
    | Screen Shot 2022-08-25 at 09 46 02 | Screen Shot 2022-08-25 at 09 47 14 |
2. Adds scope information to table headers and row's prefix column for better accessibility (in terms of visualization, this just bolds the ontology names)

    | Before | After |
    |:------:|:----:|
    | Screen Shot 2022-08-25 at 09 50 03 | Screen Shot 2022-08-25 at 09 50 12 |
3. Update quick access buttons
    1. put in button group
    2. add disabled buttons on missing values (e.g., when there's no publication) in order to make the width consistent in all sub-tables
    3. add appropriate roles for accessibility (not visible)

    | Before | After |
    |:------:|:----:|
    | Screen Shot 2022-08-25 at 09 54 06 | Screen Shot 2022-08-25 at 09 52 35 |
6. Make table sort buttons smaller
    | Before | After |
    |:------:|:----:|
    | Screen Shot 2022-08-25 at 09 55 29 |   Screen Shot 2022-08-25 at 09 55 37 |

8. Better handling of missing values (e.g., trackers, github links, descriptions, contact)
    | Before | After |
    |:------:|:----:|
    | Screen Shot 2022-08-25 at 10 01 11 | Screen Shot 2022-08-25 at 10 01 22 |

10. Fix handling of license when no image URL is available
11. Clean up superfluous divs in layouts
12. Fix missing display of orphaned ontologies (i.e., they weren't considered in the "hide inactive" button before, now they are). Note that MAMO and SIBO now appear.
    
    | Before | After |
    |:------:|:----:|
    | Screen Shot 2022-08-25 at 09 57 06  | Screen Shot 2022-08-25 at 09 57 19 |

13. Show inactivity type for orphaned/inactivity. Note that the labels for orphaned and inactive are now shown explicitly.

     | Before | After |
    |:------:|:----:|
    | Screen Shot 2022-08-25 at 09 57 06  | Screen Shot 2022-08-25 at 09 57 19 |

These changes are organized by commit, each which has some explanation in it to make review a bit easier, considering that the diff is pretty big and confusing for this whole PR

### CODE REVIEW

File: _includes/js/custom.js [hunk 1]
```diff
@@ -149,67 +149,101 @@ jQuery(document).ready(function() {
...
+            } else {
+                publication = `
+                        <a role="button" class="btn btn-default disabled">
+                            <span class="glyphicon glyphicon-book" aria-hidden="true"></span>
+                        </a>`;
             }
-            if (activity_status === "inactive") {
+            if (activity_status === "inactive" || activity_status === "orphaned") {
                 is_inactive = "inactive_row";
+                is_obsolete = `(${activity_status})`
```
Comment (matentzn) [inline _includes/js/custom.js:200]: What exactly does this change do?

**Comment by matentzn** [comment]
Great, thank you @cthoyt as always. Can I get a screenshot illustrating the changes achieved by this PR?

File: _includes/js/custom.js (hunk 1 above)
Comment (cthoyt) [reply _includes/js/custom.js:200]: I didn't change the variable name, but this makes the labels for orphaned and inactive show up.

| Before | After |
|:------:|:----:|
| Screen Shot 2022-08-25 at 09 57 06  | Screen Shot 2022-08-25 at 09 57 19 |

**Comment by cthoyt** [comment]
@matentzn just added screenshots with explanations for all big changes

**Review by matentzn — APPROVED** [review]
Another excellent contribution and I cant tell you how much I appreciate your PR description. Thank you!

**Pull request merged**